.github/ISSUE_TEMPLATE/flaky-test.md

```
@@ -0,0 +1,83 @@
+---
+name: Flaky Test
+about: Automated issue for flaky test detection
+title: '[FLAKY TEST] {{test_name}} - Intermittent failures detected'
+labels: ['flaky-test', 'test-stability', 'needs-investigation']
+assignees: ''
+---
+
+## 🔄 Flaky Test Detected
+
+**Auto-generated issue for flaky test behavior detected in CI/CD pipeline**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)